From d08b1af4bd793d19bf9711801bd6be8b507d18cf Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Fri, 15 Jun 2018 18:55:03 +0800
Subject: [PATCH] input: touchscreen: gt1x: Add missing power on in resume

Add missing power on to pair with the power off in suspend.

Change-Id: I676b57dd8ba9aff108a3b57c4350d272d54ca996
Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 drivers/input/touchscreen/gt1x/gt1x_generic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/input/touchscreen/gt1x/gt1x_generic.c b/drivers/input/touchscreen/gt1x/gt1x_generic.c
index 71b589debc97..fc7aa021a523 100644
--- a/drivers/input/touchscreen/gt1x/gt1x_generic.c
+++ b/drivers/input/touchscreen/gt1x/gt1x_generic.c
@@ -985,6 +985,7 @@ static s32 gt1x_wakeup_sleep(void)
 	gt1x_irq_disable();
 
 #if GTP_POWER_CTRL_SLEEP	/* power manager unit control the procedure */
+	gt1x_power_switch(SWITCH_ON);
 	gt1x_power_reset();
 	GTP_INFO("Wakeup by poweron");
 	return 0;
-- 
2.35.3

